modules/new_profile/php/new_profile.class.inc

```diff
@@ -205,13 +207,11 @@ class New_Profile extends \NDB_Form
         $user         = $factory->user();
         $db           = $factory->database();
 
-        if (empty($values['dobDate'])) {
-            $errors['dobDate'] = 'Date of Birth field is required.';
-        }
-        if (empty($values['dobDateConfirm'])) {
-            $errors['dobDateConfirm'] = 'Confirm Date of Birth field is required.';
-        }
-        if ($values['dobDate'] != $values['dobDateConfirm']) {
+        // Validate DOB fields
+        if (isset($values['dobDate'])
+            && isset($values['dobDateConfirm'])
```

Comment:
FYI you can do `isset($values['dobDate'],$values['dobDateConfirm'])`

isset() accepts multiple parameters